WordPress/src/main/java/org/wordpress/android/ui/people/PeopleInviteFragment.java

```diff
@@ -423,6 +431,10 @@ private void checkAndSend() {
             return;
         }
 
+        if (!NetworkUtils.checkConnection(getActivity())) {
+            return;
```

Comment:
The implementation of `NetworkUtils.checkConnection()` actually shows a Toast when there is no network 👍